skore-remote-project/src/skore_remote_project/project/widget.py

```diff
@@ -380,6 +382,9 @@ def _add_jitter_to_categorical(
         np.ndarray
             Array of encoded categorical values with jitter applied.
         """
+        if categorical_series.cat.categories.size == 1:
+            return categorical_series.cat.codes.to_numpy()
```

Comment:
exit early when we have a single category, otherwise the results is really weird :)